spec/models/survey_spec.rb

```diff
@@ -1,3 +1,31 @@
 require 'rails_helper'
 
-describe Survey do end
+describe Survey do
+  describe 'formatted_date' do
+    it 'returns the formatted date' do
+      date = DateTime.current
+      survey = create :survey, date: date
+      expect(survey.formatted_date).to eql date.strftime('%m/%d/%Y')
```

Comment:
To clarify, my concern wasn't about readability or clarity, but rather about the test code echoing the same logic as the code it's testing. In general that's something to avoid, especially if you're the person writing both, since it can be easy to make the same mistake twice, for instance.

The change you made addresses that concern also, nice work